package.json

```diff
@@ -117,64 +119,48 @@
         "codelyzer": "6.0.2",
         "copy-webpack-plugin": "9.0.1",
         "cross-env": "7.0.3",
-        "css-loader": "6.2.0",
+        "css-loader": "6.3.0",
         "eslint": "7.32.0",
         "eslint-config-prettier": "8.3.0",
         "eslint-plugin-chai-friendly": "0.7.2",
-        "eslint-plugin-prettier": "3.4.0",
+        "eslint-plugin-prettier": "4.0.0",
         "eslint-webpack-plugin": "3.0.1",
-        "file-loader": "6.2.0",
-        "fork-ts-checker-webpack-plugin": "6.3.1",
-        "friendly-errors-webpack-plugin": "1.7.0",
-        "html-loader": "2.1.2",
-        "html-webpack-plugin": "5.3.2",
-        "husky": "7.0.1",
-        "jest": "26.6.3",
+        "folder-hash": "4.0.1",
+        "husky": "7.0.2",
+        "jest": "27.2.0",
         "jest-canvas-mock": "2.3.1",
         "jest-date-mock": "1.0.8",
-        "jest-junit": "12.1.0",
-        "jest-preset-angular": "8.4.0",
+        "jest-junit": "12.2.0",
+        "jest-preset-angular": "10.0.0",
         "jest-sinon": "1.0.4",
         "lint-staged": "11.1.2",
         "merge-jsons-webpack-plugin": "2.0.0-alpha",
-        "mini-css-extract-plugin": "2.2.0",
-        "moment-locales-webpack-plugin": "1.2.0",
+        "mini-css-extract-plugin": "2.3.0",
         "ng-mocks": "12.3.1",
         "postcss": "8.3.6",
         "postcss-loader": "6.1.1",
-        "prettier": "2.3.2",
+        "prettier": "2.4.0",
         "reflect-metadata": "0.1.13",
         "rimraf": "3.0.2",
-        "sass": "1.37.5",
-        "sass-loader": "12.1.0",
-        "simple-progress-webpack-plugin": "2.0.0",
+        "sass": "1.41.0",
         "sinon": "11.1.2",
         "sinon-chai": "3.7.0",
-        "stream-browserify": "3.0.0",
-        "style-loader": "3.2.1",
-        "thread-loader": "3.0.4",
-        "to-string-loader": "1.1.6",
-        "ts-jest": "26.5.6",
-        "ts-node": "10.1.0",
-        "tsconfig-paths": "3.10.1",
+        "ts-jest": "27.0.5",
+        "ts-node": "10.2.1",
+        "tsconfig-paths": "3.11.0",
         "tslint": "6.1.3",
         "tslint-config-prettier": "1.18.0",
         "tslint-no-unused-expression-chai": "0.1.4",
         "typescript": "4.3.5",
-        "url-loader": "4.1.1",
         "uuid": "8.3.2",
-        "weak-napi": "2.0.2",
-        "webpack": "5.49.0",
+        "webpack": "5.52.1",
```

Comment:
Do we need webpack anymore since we replaced everything with the angular commands?